task/jira-get-ticket-status/0.1/jira-get-ticket-status.yaml

```diff
@@ -0,0 +1,81 @@
+apiVersion: tekton.dev/v1beta1
```

Comment:
can we use latest apiVersion ? ie `tekton.dev/v1`

Reply:
done!